src/generator/AutoRest.Go/rakefile.rb

```diff
@@ -13,18 +16,18 @@
 	compute: {version: "2016-03-30"},
 	# containerservice: {version: "2016-03-30", swagger: "containerservice"},
 	# commerce: {version: "2015-06-01-preview"},
-	# datalake_analytics: {
-	# 	account: {version: "2015-10-01-preview"},
-	# 	catalog: {version: "2015-10-01-preview"},
-	# 	job:  {version: "2016-03-20-preview"}
-	# },
+	datalake_analytics: {
+		account: {version: "2015-10-01-preview"},
+		# catalog: {version: "2016-06-01-preview"},
```

Comment:
job and catalog are not azure resource manager api of DataLake. They belong to the data plane of the DataLake resource provider.